src/main/java/com/clickhouse/kafka/connect/sink/db/ClickHouseWriter.java

```diff
@@ -95,6 +95,9 @@ public void setBinary(boolean binary) {
         isBinary = binary;
     }
 
+    private String escapeTopicName(String topic) {
+        return String.format("`%s`", topic);
```

Comment:
Why does the table name escaping rules leak outside? The table name is be escaped on `L360` https://github.com/ClickHouse/clickhouse-kafka-connect/pull/39/files#diff-3180bd2c71e5c2af1a2aafe6ba0d52a50da1e97cefc6528911a35dda50c46ed0R360 isn't it?

Reply:
We are escaping the topic name since when we get it from Map, the keys are already escaped. I planning to make a bigger refactor when adding also the database name to the map